File: src/Administrador.java
public class Administrador {
    private int idAdmin;
    private String nombreAdmin;
    private String correoAdmin;
    private String contrasena; // Por que el admin debe logearse.
    private boolean estadoConexion = false;

    public Administrador(int idAdmin, String nombreAdmin, String correoAdmin, String contrasena){
        this.idAdmin = idAdmin;
        this.nombreAdmin = nombreAdmin;
        this.correoAdmin = correoAdmin;
        this.contrasena = contrasena;
    }
    // ========Getters========
    public int getIdAdmin() {
        return idAdmin;
    }

    public String getNombreAdmin() {
        return nombreAdmin;
    }

    public String getCorreoAdmin() {
        return correoAdmin;
    }

    public String getContrasena() {
        return contrasena;
    }

    public boolean isConectado() {
        return estadoConexion;
    }
    // ========Setters========
    public void login(){
        estadoConexion = true;
    }
}

File: src/Administrador.dart
class Administrador {
    int idAdmin;
    String nombreAdmin;
    String correoAdmin;
    String contrasena; // Por que el admin debe logearse.
    bool estadoConexion = false;

    Administrador({
        required this.idAdmin,
                required this.nombreAdmin,
                required this.correoAdmin,
                required this.contrasena,
    });

    // ========Getters========
    int get getIdAdmin => idAdmin;

    String get getNombreAdmin => nombreAdmin;

    String get getCorreoAdmin => correoAdmin;

    String get getContrasena => contrasena;

    bool get isConectado => estadoConexion;

    // ========Setters========
    void login() {
        estadoConexion = true;
    }
}
